.changeset/tough-eagles-visit.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/select-utils': patch
+---
+
+Migrate to typescript
```

Comment:
Let's also mention that `react-select` got updated to v5

Reply:
This is already done in the react-select PR